src/client/Microsoft.Identity.Client/Internal/Requests/DeviceCodeRequest.cs

```diff
@@ -103,6 +103,7 @@ await Task.Delay(TimeSpan.FromSeconds(deviceCodeResult.Interval), cancellationTo
         {
             var dict = new Dictionary<string, string>
             {
+                [OAuth2Parameter.ClientInfo] = "1",
```

Comment:
This used to be here in the token client for all of our requests.
https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/31471d957e278465bf31e66a0ec6e6c1690f9f4b/src/client/Microsoft.Identity.Client/OAuth2/TokenClient.cs#L133

Any idea why it was removed? Should it be put back instead of adding it to the different requests?

Reply:
I'll try to separate out a commit for this. We should not send this to ALL our requests, in particular for the perf critical client_credential request.